lib/worker.js

```diff
@@ -60,8 +70,13 @@ function Worker (opts) {
 
   // put all opts on this
   assign(this, opts);
+  this.log.info({ queue: this.queue, job: this.job }, 'Worker created');
 
-  log.info({ queue: this.queue, job: this.job }, 'Worker created');
+  // Ensure that the `msTimeout` option is valid
+  this.msTimeout = Math.abs(parseInt(this.msTimeout));
+  if (!isNumber(this.msTimeout)) {
+    throw new Error('Provided `msTimeout` is not an integer');
+  }
```

Comment:
I don't see a unit test specifying that `Math.abs` is here and preventing negative numbers... I'm curious about this extra check...